Language/Variables/Implicitly_Induced_Getters_and_Setters/final_A03_t03.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2011, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A non-local variable introduces a setter iff it does not have the
+/// modifier final or the modifier const, or it is late and final, but does not
+/// have an initializing expression.
```

Comment:
Similar to Language/Variables/Implicitly_Induced_Getters_and_Setters/final_A03_t01.dart.